locales/en-US/app.ftl

```diff
@@ -1,8 +1,69 @@
-# Please sort alphabetically by (module name, component name). For each module,
+# Please sort alphabetically by (component name).
 # keep strings in order of appearance.
 
-intro = Web app for Firefox performance analysis
+## The following feature names must be treated as a brand. They cannot be translated
+
+-firefox-brand-name = Firefox
+-profiler-brand-name = Firefox Profiler
+
+## Translation Strings
+
+AppHeader--app-header = <header>{ -profiler-brand-name }</header> — <subheader>Web app for { -firefox-brand-name } performance analysis</subheader>
+AppHeader--github-icon =
+    .title = Go to our git repository (this opens in a new window)
+
+FooterLinks--legal = Legal
+FooterLinks--Privacy = Privacy
+FooterLinks--Cookies = Cookies
+
+Home--profiler-motto = Capture a performance profile. Analyze it. Share it. Make the web faster.
+Home--documentation-button = Documentation
+Home--instructions-title = How to view and record profiles
+Home--instructions-content = 
+    Recording performance profiles requires <a>{ -firefox-brand-name }</a>.
+    However, existing profiles can be viewed in any modern browser.
+
+Home--additional-content-title = Load existing profiles
+Home--additional-content-content = You can <strong>drag and drop</strong> a profile file here to load it, or:
+Home--upload-from-file-input-button = Load a profile from file
+Home--upload-from-url-button = Load a profile from a URL
+Home--compare-recordings-info = You can also compare recordings. <a>Open the comparing interface.</a>
+Home--recent-uploaded-recordings-title = Recent uploaded recordings
+Home--record-instructions = 
+    To start profiling, click on the profiling button, or use the
+    keyboard shortcuts. The icon is blue when a profile is recording. 
+    Hit <kbd>Capture Profile</kbd> to load the data into profiler.firefox.com.
+
+Home--record-instructions-start-stop = Stop and start profiling
+Home--record-instructions-capture-load = Capture and load profile
+Home--load-from-url-submit-button =
+    .value = Load
+
+# This string is used on the tooltip of the published profile links
+# Variables:
+#   $smallProfileName (String) - Shortened name for the published Profile
+ListOfPublishedProfiles--publishedProfilesLink =
+    .title = Click here to load profile { $smallProfileName }
+
+ListOfPublishedProfiles--uploaded-profile-information-list-empty = No profile has been uploaded yet!
+ListOfPublishedProfiles--uploaded-profile-information-label = See and manage all your recordings ({ $profilesRestCount } more)
+
+# These strings are used as placable depending on the value of the variable, $uploadedProfileCount
+# Variables:
+#   $uploadedProfileCount (Number) - Toatal numbers of the uploaded profiles
+ListOfPublishedProfiles--uploaded-profile-information-list = 
+    { $uploadedProfileCount ->
+        [one] Manage this recording
+       *[other] Manage these recordings
+    }
+
+ProfileRootMessage--title = { -profiler-brand-name }
```

Comment:
Maybe add a comment explaining that this doesn't likely need to be translated. @flodolo do we add such comments usually or is that obvious?

Reply:
No need, that will raise warnings if changed, and it's a syntax localizers are familiar with.